templates/firstboot-nvme.yaml.j2

```diff
@@ -33,6 +33,11 @@ resources:
             if [[ $(hostname) == *$FORMAT* ]] ; then
               sed "s/^\(GRUB_CMDLINE_LINUX=\".*\)\"/\1 $KERNEL_ARGS\"/g" -i /etc/default/grub ;
               grub2-mkconfig -o /etc/grub2.cfg
+              mkdir -p /var/lib/nova
+              mkfs -t xfs -f /dev/nvme0n1
+              mount /dev/nvme0n1 /var/lib/nova
+              chmod a+w /var/lib/nova
+              echo UUID=`sudo blkid -s UUID -o value /dev/nvme0n1` /var/lib/nova xfs discard,defaults,nofail 0 2 | sudo tee -a /etc/fstab
```

Comment:
Do you plan to keep the nvme path hardcoded? Would be good to parameterize inside of the `mount_nvme` dictionary.